addons/website_forum/models/forum.py

```diff
@@ -126,6 +126,8 @@ def _get_default_faq(self):
     karma_editor = fields.Integer(string='Editor Features: image and links',
                                   default=30, oldname='karma_editor_link_files')
     karma_user_bio = fields.Integer(string='Display detailed user biography', default=750)
+    karma_post = fields.Integer(string='Ask questions without validation', default=100)
```

Comment:
When you create a new user account on the forum, you have 0 as karma. When your email is validated, you have "karma_ask" karma, and you can ask a question on the forum, but until you don't have "karma_question", your question must be "accepted" by a moderator ("karma_moderator") to be displayed to all users ("trending", "newest", "search", ...).
Karma_answer is just the karma needed to answer a question. Those answers don't need to be "accepted" by a moderator (if you have "karma_answer", you can answer a question, and the answer is automatically displayed to all the users. We could change this if needed, but the idea is to don't have too many posts to moderate).